bluepyopt/ephys/simulators.py

```diff
@@ -16,11 +16,14 @@ def __init__(self, dt=None, cvode_active=True, cvode_minstep=None):
 
         import imp
         import ctypes
-
-        hoc_so = os.path.join(imp.find_module('neuron')[1] + '/hoc.so')
-
-        nrndll = ctypes.cdll[hoc_so]
-        ctypes.c_int.in_dll(nrndll, 'nrn_nobanner_').value = 1
+        import platform
+
+        if platform.system()!='Windows':
+            # hoc.so does not exist on NEURON Windows
```

Comment:
Mmm, we would need a construct that works after starting python. Typically the bluepyopt scripts are not run from an 'nrniv -python' pompt.